doc/zypper.8.txt

```diff
@@ -777,6 +777,27 @@ Query Commands
 	*--supplements*::
 		Search for packages which supplement the search strings.
 
+	*--provides-pkg*::
+		Search for all packages that provide any of the provides of the package(s) matched by the input parameters.
+
+	*--requires-pkg*::
+		Search for all packages that require any of the provides of the package(s) matched by the input parameters.
+
+	*--recommends-pkg*::
+		Search for all packages that recommends any of the provides of the package(s) matched by the input parameters.
```

Comment:
`Search for all packages that recommend`, without `s`?
Same in the descriptions below.

Reply:
Done